modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/services/AccountsService.java

```diff
@@ -114,6 +122,76 @@ public void toggle(UUID accId, boolean adminFlag) {
         }
     }
 
+    /**
+     * @param oldVal Old value.
+     * @param newVal New value.
+     * @return {@code true} if value should be updated.
+     */
+    private boolean changed(String oldVal, String newVal) {
+        return !F.isEmpty(newVal) && !oldVal.equals(newVal);
+    }
+
+    /**
+     * Save user.
+     *
+     * @param changes Changes to apply to user.
+     */
+    public void save(JsonObject changes) {
```

Comment:
Why we use JsonObject instead of POJO class?

Reply:
Fixed.